pkg/dsdk/connection.go

```diff
@@ -66,7 +66,7 @@ type ApiErrorResponse struct {
 	ConnInfo     map[string]string `json:"connInfo,omitempty"`
 	ClientId     string            `json:"client_id,omitempty"`
 	ClientType   string            `json:"client_type,omitempty"`
-	Id           string            `json:"api_req_id,omitempty"`
+	Id           int               `json:"api_req_id,omitempty"`
```

Comment:
Is there a reason why 'Id' is declared as int now? Any operations that needs 'Id' needs to be a integer? The 'ClientId' before this is declared a string. Would be confusing to have one of them String type and the Int type.

Reply:
I think ClientID should also be int, you are right. Code and Http already are above. I hit this during testing, the json test file (remote_provider.json) is being mapped to this struct and it errors out otherwise.